src/aws-cpp-sdk-core/source/client/AWSClient.cpp

```diff
@@ -359,6 +372,9 @@ HttpResponseOutcome AWSClient::AttemptExhaustively(const Aws::Http::URI& uri,
         httpRequest->SetHeaderValue(Http::SDK_REQUEST_HEADER, requestInfo);
         Aws::Monitoring::OnRequestRetry(this->GetServiceClientName(), request.GetServiceRequestName(), httpRequest, contexts);
     }
+    auto meter = m_telemetryProvider->getMeter(this->GetServiceClientName(), {});
+    auto counter = meter->CreateCounter("smithy.client.attempts", "n", "");
```

Comment:
really, create string literals static variables somewhere with more hinting names, like what "n" mean in this context?